_aggregations/metric/average.md

```diff
@@ -9,9 +9,20 @@ redirect_from:
 
 # Average aggregations
```

Comment:
I've omitted scripts and histogram fields.

Scripts because they're covered in detail in Scripted Metric Aggregation.

Histogram fields because they don't seem to be supported anymore. Please confirm.